source/settings.json

```diff
@@ -906,7 +923,7 @@
     "step_background_color": "{{ background_color }}",
     "step_inactive_border_color": "opacify({{ border_color }}, .25)",
     "step_primary_font_weight": "normal",
-    "step_header_color": "{{ text_color }}",
+    "step_header_color": "{{ primary_text_color }}",
```

Comment:
@nickendle it's correct I have to change these settings here too right?

Reply:
@tmchow the checkout settings are (mostly) for legacy checkout, i think the only ones we need to worry about are the header background/text and the width/height